var/spack/repos/builtin/packages/blast-plus/package.py

```diff
@@ -57,7 +58,16 @@ def url_for_version(self, version):
     # On the other hand, the `find` command is broken and there
     # aren't any .svn dirs in the tree, so I've updated their patch
     # to just comment out the block.
-    patch('blast-make-fix2.5.0.diff', when="@:2.6.0")
+    patch('blast-make-fix2.5.0.diff', when="@2.5.0:2.6.0")
+
+    # See https://github.com/Homebrew/homebrew-science/\
+    # issues/2337#issuecomment-170011511
```

Comment:
You can put this on a single line if you want. We ignore lines containing `https?:` when running flake8 line length checks.